### DISCUSSION TITLE
I won a block, and ask for support and you close the case with no solution

### DISCUSSION THREAD

**Discussion by jealalri** [Chia-Network/chia-blockchain#8563] [Support]
I won a block in space pool, i got this 
Fecha
9/20/21, 10:47 PM
Altura
887426
Hash
1af00ac572a09975433d8bec790998390e93814646fd13c6d1937fad0046c4b9
My farmer Id is: 
0xd86a7d4ff00ca5545c375093756812d094815db40529738fc9f68c6bf8912eb5
the reward wallet where they are suppose to send the token is: xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p
and what you do is close the case

**Comment by loppefaaret** [comment]
please use the discussion we already have open for this:
#8542

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?
